prow/entrypoint/run.go

```diff
@@ -83,18 +88,33 @@ func (o Options) Run() error {
 		commandErr = err
 	case <-time.After(timeout):
 		logrus.Errorf("Process did not finish before %s timeout", timeout)
-		if err := command.Process.Kill(); err != nil {
-			logrus.WithError(err).Error("Could not kill process after timeout")
+		cancelled = true
+		if err := command.Process.Signal(os.Interrupt); err != nil {
```

Comment:
How come? SIGINT (ctrl-c) seems more right to me than `kill pid`.

Reply:
SGTM. I was just wondering if this was a mistake because of the PR title. 
SIGINT does seem more appropriate, but something to keep in mind is that kubernetes sends SIGTERM to containers when requesting that a Pod terminate before sending SIGKILL. Do we care about remaining consistent with the signals kubernetes sends?

Reply:
I think SIGTERM is an implementation detail of the Docker container runtime, it's not the kubelet making that call. `SIGINT` seems more appropriate? IDK, if people are expecting `SIGTERM` today we can change it, or do this now and see what workloads end up needing in reality?

Reply:
I would also err on the side of `SIGINT` because if we are running effectively shell scripts inside of repos to test, those should act well in the case that a dev hits `Ctrl+C`, sending `SIGINT`. Want to mimic that behavior here. Dev won't usually send `SIGTERM`